Controller/Customer/Index.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\ProductFAQ\Controller\Customer;
+
+use Inchoo\ProductFAQ\Api\FaqRepositoryInterface;
+use Magento\Customer\Model\Session;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Registry;
+
+class Index extends Action
+{
+    protected $faqRepository;
```

Comment:
stavljaj DocBlock comment na sve property-e

primjer:
```suggestion
    /**
     * @var FaqRepositoryInterface
     */
    protected $faqRepository;
```

Reply:
DocBlock dodan gdje nedostaje (property and methods)